src/IntegratedExperts/BehatScreenshotExtension/Context/ScreenshotContext.php

```diff
@@ -86,7 +86,6 @@ public function beforeScenarioInit(BeforeScenarioScope $scope)
                 if (!$driver->isStarted()) {
                     $driver->start();
                 }
-                $this->getSession()->resizeWindow(1440, 900, 'current');
```

Comment:
@xurizaemon 
Can we please get this reverted? I'm not sure what is the reason to remove this.

This was set as one of the most widely-used full-screen sizes.

The added step definition will override the value set in before scenario.

Reply:
Sure! (This was WIP towards configuring the default screen size from behat.yml, but that can wait.)

Reply:
Done :)